src/Products/PloneMeeting/model/adaptations.py

```
@@ -164,6 +164,13 @@ def change_transition_new_state_id(wf_id, transition_id, new_state_id):
         transition_id, new_state_id, wf_id))
 
 
+def _workflow_object_id(value):
+    """Return a Zope workflow object id."""
+    if isinstance(value, unicode):
```

Ruff (0.15.10)

[error] Undefined name `unicode`

(F821)